README.md

```diff
@@ -204,10 +204,10 @@ console.log(shoukaku.getIdealNode());
  Option | Type | Description
 --------|------|------------
 resume | boolean | Whether to resume a connection on disconnect to Lavalink (Server Side) (Note: DOES NOT RESUME WHEN THE LAVALINK SERVER DIES) |
-resumeKey | string | Resume key for Lavalink |
+sessionId | string | Session ID for lavalink to resume |
```

Comment:
I think this is not user a user input. Based on https://github.com/lavalink-devs/Lavalink/blob/master/IMPLEMENTATION.md#ready-op
sessionId is received when we connect to lavalink, hence in order to resume, we need to send the same session id we cached